crates/swc/tests/tsc-references/additionOperatorWithAnyAndEveryType_es2015.2.minified.js

```diff
@@ -1,7 +1,12 @@
-var E, E, M, a, b;
+var E, E, M, a, b, c, d, e;
```

Comment:
Nevermind about this. This file is not a good input because the testing system of swc does not support `@filename` directives of tsc test suite.